dags/openshift_nightlies/config/benchmarks/data-plane.json

```diff
@@ -25,6 +25,29 @@
                 "WORKLOAD": "pod2svc"
             }
         },
+        {
+            "name": "router",
+            "workload": "router-perf-v2",
+            "command": "./ingress-performance.sh",
+            "env": {
+                "THROUGHPUT_TOLERANCE": "10",
+                "LATENCY_TOLERANCE": "10",
+                "ENGINE": "local",
+                "COMPARE": "false",
+                "EMAIL_ID_FOR_RESULTS_SHEET": "msheth@redhat.com",
+                "GSHEET_KEY_LOCATION": "/tmp/key.json"
+            }
+        },
+        {
+            "name": "scale-27",
+            "workload": "scale-perf",
+            "command": "./run_scale_fromgit.sh",
+            "env": {
+                "SCALE": "27",
+                "METADATA_COLLECTION": "true",
+                "WORKLOAD_NODE_ROLE": "workload"
+            }
+        },
```

Comment:
small doubt, why run router before and also after scaling?

Reply:
Ingress-performance with 3 nodes and with 27 nodes ;)